MiniSpace.Services.Reactions/src/MiniSpace.Services.Reactions.Api/Program.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Convey;
+using Convey.Logging;
+using Convey.Types;
+using Convey.WebApi;
+using Convey.WebApi.CQRS;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+using MiniSpace.Services.Reactions.Application;
+using MiniSpace.Services.Reactions.Application.Commands;
+using MiniSpace.Services.Reactions.Application.Dto;
+using MiniSpace.Services.Reactions.Application.Queries;
+using MiniSpace.Services.Reactions.Core.Entities;
+using MiniSpace.Services.Reactions.Infrastructure;
+
+namespace MiniSpace.Services.Reactions.Api
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+            => await WebHost.CreateDefaultBuilder(args)
+                .ConfigureServices(services => services
+                    .AddConvey()
+                    .AddWebApi()
+                    .AddApplication()
+                    .AddInfrastructure()
+                    .Build())
+                .Configure(app => app
+                    .UseInfrastructure()
+                    .UseDispatcherEndpoints(endpoints => endpoints
+                        .Get("", ctx => ctx.Response.WriteAsync(ctx.RequestServices.GetService<AppOptions>().Name))
+
+                        // get reactions: content needed
+                        .Get<GetReactions, IEnumerable<ReactionDto>>("reactions")
+
+                        // reaction summary: content needed
+                        .Get<GetReactionsSummary, (int NumberOfReactions, ReactionType DominantReaction)>("reactions/summary")

```

Comment:
It would be better if this endpoint returned ReactionsSummaryDto with such fields instead of a tuple